frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -31,6 +31,18 @@ provided by Console to its dynamic plugins.
 _[OpenShift Console Dynamic Plugins feature page][console-doc-feature-page]_ - A high-level overview of
 dynamic plugins in relation to OLM operators and cluster administration.
 
+## Plugin project references
+
+If you're new to dynamic plugins, we suggest to clone the [Console plugin template][console-plugin-template]
+repo and try to build and run it using a [local Console Bridge server](#local-plugin-development).
+
+You can also take a look at [Console demo plugin][console-demo-plugin] located in the Console repo,
+however note that this plugin is primarily meant for testing the current Console plugin SDK features.
+
+Here is a list of real world dynamic plugins that may serve as a reference point:
```

Comment:
I probably wouldn't call this a "world real" plugin since it's not a functioning CRD, more an example plugin that uses the example CRD from the k8s doc. I'd label it as a sample instead.

Reply:
I like the idea of the real world plugins, but for that leats rather create a separate list and lets put there kubevirt, monitoring and other plugins which are currently running in console.